lib/alchemy/resource.rb

```diff
@@ -160,7 +160,13 @@ def editable_attributes
     # For now it only uses string type columns
     #
     def searchable_attributes
-      self.attributes.select { |a| a[:type].to_sym == :string }
+      attributes.select { |a| a[:type].to_sym == :string && !a.has_key?(:relation) } +
+      attributes.select { |a| a.has_key?(:relation) }.map do |a|
```

Comment:
Style/MultilineOperationIndentation: Use 2 (not 0) spaces for indenting an expression spanning multiple lines.